samples/dnn/caffe_googlenet.cpp

```diff
@@ -169,7 +171,11 @@ int main(int argc, char **argv)
     //! [Gather output]
 
     //! [Print results]
-    std::vector<String> classNames = readClassNames();
+    std::vector<String> classNames;
+    if (classNameFile.empty())
+        classNames = readClassNames();
```

Comment:
Does this condition really pass? I mean there is default value for this parameter, so it should not be empty.